pages/1.13/cli/command-reference/dcos-security/dcos-security-cluster/dcos-security-cluster-directory/dcos-security-cluster-directory-get_config/index.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: layout.pug
+navigationTitle:  dcos security cluster directory get_config
+title: dcos security cluster directory get_config
+menuWeight: 32
+excerpt: Retrieving an LDAP configuration
+enterprise: true
+---
+
+# Description
+
+The `dcos security cluster directory get_config` command will retrieve the current LDAP configuration.
```

Comment:
why not name this dcos security cluster ldap get_config, or even dcos security ldap get_config

Reply:
This is a good question for the Security team, I believe this is out of the scope of this PR. Can you please ping the PM of the Security team regarding this so that we can move this subcommand and start deprecating `dcos security cluster directory get_config` in 1.14?